src/view-mixin.js

```diff
@@ -138,6 +118,9 @@ var AbstractView = Backbone.View.extend({
   // Overriding Backbone.View's `delegateEvents` to handle
   // `events` and `triggers`
   delegateEvents: function(eventsArg) {
+    // proxy behavior $el to the view's $el.
+    _.invoke(this._behaviors, 'proxyViewProperties', this);
```

Comment:
Moved this from `setElement` as it removed the necessity for another protocol call, but will still occur in the same order.